src/routes/subscriptions.rs

```diff
@@ -263,11 +184,7 @@ pub async fn insert_subscriber(
         Utc::now()
     )
     .execute(transaction)
-    .await
-    .map_err(|e| {
-        tracing::error!("Failed to execute query: {:?}", e);
-        e
-    })?;
+    .await?;
```

Comment:
Doesnt make sense to log the same error twice (although I know `axum` logs an error before returning the response. Idk how to capture the error there yet)